frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java

```diff
@@ -357,6 +359,14 @@ public Connector getListener(ConnectorType connectorType) {
         return Connector.parse(binding, connectorType);
     }
 
+    public InetAddress getGRPCAddress(ConnectorType connectorType) throws UnknownHostException {
+        if (connectorType == ConnectorType.MANAGEMENT_CONNECTOR) {
+            return InetAddress.getByName(prop.getProperty(TS_GRPC_MANAGEMENT_ADDRESS, "127.0.0.1"));
+        } else {
```

Comment:
should we be more explicit in the if else that the second clause is only for inference?